### PR TITLE
Mock post pitches endpoint

### PR DESCRIPTION
**Pull request by osziszab**

### CODE REVIEW

File: ASP.NET-Core-Webapp/Controllers/PitchController.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            pitches.Add(new Pitch() { BadgeName = "Java pro", OldLevel = 2, PitchedLevel = 3, PitchMessage = "Hello World! My English is bloody gorgeous.", Holders = { "balazs.jozsef", "benedek.vamosi", "balazs.barna" } });
+            pitches.Add(new Pitch() { BadgeName = "C# pro", OldLevel = 2, PitchedLevel = 3, PitchMessage = "Hello World! My English is bloody gorgeous.", Holders = { "balazs.jozsef", "benedek.vamosi", "balazs.barna" } });
+        }
+
+        [HttpPost("pitches")]
+        public IActionResult CreateNewPitch(Pitch newPitch)
+        {        
+            if (!pitches.Exists(e => e.BadgeName.Equals(newPitch.BadgeName)) && !newPitch.Equals(null)) {
+                pitches.Add(newPitch);               
+                return Created("",new { message = "Success" });
```
Comment (fabianzs) [inline ASP.NET-Core-Webapp/Controllers/PitchController.cs:30]: Is "" necessary?

**Review by fabianzs — DISMISSED** [review]
Seems OK to me!

**Review by szirmi — DISMISSED** [review]

File: ASP.NET-Core-Webapp/Controllers/PitchController.cs (hunk 1 above)
Comment (osziszab) [reply ASP.NET-Core-Webapp/Controllers/PitchController.cs:30]: Yap

**Review by fabianzs — APPROVED** [review]
Ok!

**Pull request merged**